nodewatcher/modules/defaults/network_profile/models.py

```diff
@@ -36,5 +36,6 @@ class RegistryMeta:
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('wifi-uplink', _("Use a wireless interface for uplink")))
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('no-wifi-ap', _("Disable wireless AP")))
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('no-wifi-mesh', _("Disable wireless mesh")))
+registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('802.11s-wifi-mesh', _("Enable 802.11s wireless mesh")))
```

Comment:
I think a better name would be `wifi-mesh-80211s`. And the description more like `"Use 802.11s for meshing"`.